Source/Interfaces/Preferences.xib

```diff
@@ -755,10 +755,43 @@ Gw
                         </constraints>
                     </view>
                 </box>
+                <textField verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="gcQ-a7-Kqd">
+                    <rect key="frame" x="18" y="18" width="154" height="20"/>
+                    <constraints>
+                        <constraint firstAttribute="width" constant="150" id="UXx-vH-pTm"/>
+                        <constraint firstAttribute="height" constant="20" id="cZz-2P-Xzw"/>
+                    </constraints>
+                    <textFieldCell key="cell" scrollable="YES" lineBreakMode="clipping" sendsActionOnEndEditing="YES" alignment="right" title="Analytics" id="6mk-ml-lFe">
+                        <font key="font" metaFont="system"/>
+                        <color key="textColor" name="controlTextColor" catalog="System" colorSpace="catalog"/>
+                        <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
+                    </textFieldCell>
+                </textField>
+                <button translatesAutoresizingMaskIntoConstraints="NO" id="2aR-JM-Lbr">
+                    <rect key="frame" x="178" y="16" width="342" height="24"/>
+                    <constraints>
+                        <constraint firstAttribute="height" constant="22" id="f9S-3k-CY3"/>
+                    </constraints>
+                    <buttonCell key="cell" type="check" title="Send application usage analytics" bezelStyle="regularSquare" imagePosition="left" alignment="left" inset="2" id="mZ1-oa-LuT">
```

Comment:
Slight tweak on language:
"Share anonymized usage analytics with developers" or something like that

Reply:
Updated at ab631f83889c973d605395481d64e3c23d6ba853